cert/README.txt

```diff
@@ -0,0 +1,20 @@
+This is a Test Ceritificate Authority, only to be used for testing.
```

Comment:
Why not generate the certificates at build time? (basically put these commands in the Makefile).

Reply:
It seems a bit more safe and predictable to have pre-generated keys and certificates. Then we also know that we're testing with the same input each time.

Reply:
I'm still not convinced. I think we can safely assume that we have a working OpenSSL implementation and it would reduce the amount of opaque data committed into the repository.
But it's not a strong opinion, I'm OK with the current proposal.